website/content/api-docs/secret/aws.mdx

```diff
@@ -670,6 +670,10 @@ This endpoint creates or updates static role definitions. A static role is a 1-t
 with an AWS IAM User, which will be adopted and managed by Vault, including rotating it according
 to the configured `rotation_period`.
 
+In addition, this endpoint supports cross account management.
+Vault will use the credentials obtained by assuming roles in another AWS account to perform AWS operations.
+Make sure that the IAM user or role configured in Vault has the necessary permissions to assume those roles.
```

Comment:
```suggestion
Make sure that the IAM role configured in Vault has the necessary permissions to manage the IAM user within the target account.
```
Please let me know if I'm mistaken, but I think this is the information you are trying to say? I think the current wording is a bit confusing.